content/d-sgov/d-sgov-právní-akty/d-sgov-právní-akty-glosář.ttl

```diff
@@ -128,7 +129,7 @@ d-sgov-právní-akty-pojem:citace-názvu-znění-právního-aktu a skos:Concept;
   skos:inScheme d-sgov-právní-akty:glosář;
   skos:prefLabel "citace názvu znění právního aktu"@cs .
 
-d-sgov-právní-akty-pojem:zkrácený-název-právního-aktu a skos:Concept;
+d-sgov-právní-akty-pojem:zkrácený-název-znění-právního-aktu a skos:Concept;
```

Comment:
tohle možná budeme nakonec zase přesouvat k právnímu aktu, ale zatím je to správně